osgar/node.py

```diff
@@ -0,0 +1,31 @@
+"""
+  Node - processing unit
+"""
+from threading import Thread
+
+from osgar.bus import BusShutdownException
+
+
+class Node(Thread):
+    """
+       Parent classs for all processing "nodes".
+    """
+    def __init__(self, config, bus):
+        Thread.__init__(self)
```

Comment:
```suggestion
        super().__init__()
```
https://docs.python.org/3.5/library/functions.html#super